### PR TITLE
test: add .env.example file with its description to integration-tests Readme.md

### PR DESCRIPTION
**Pull request by pcheremu**
# What ❔

env.example file with Readme description to integration-tests

## Why ❔

To improve UX and avoid issues in preparing a local env 

## Checklist
- env.example
- Readme.md

- [ + ] PR title corresponds to the body of PR (we generate changelog entries from PRs).
- [ n\a ] Tests for the changes have been added / updated.
- [ + ] Documentation comments have been added / updated.

### CODE REVIEW

**Review by Romsters — APPROVED** [review]

**Review by vasyl-ivanchuk — APPROVED** [review]

File: packages/integration-tests/src/playbook/hardhat.config.ts [hunk 1]
```diff
@@ -1,13 +1,21 @@
...
 import "@matterlabs/hardhat-zksync-solc";
 import "@nomiclabs/hardhat-ethers";
+import * as fs from "fs";
+
 import { localConfig } from "../config";
+import { Wallets } from "../entities";
 
 import type { HardhatUserConfig } from "hardhat/types";
 
+const envFilePath = `${__dirname}/.env`;
```
Comment (Romsters) [inline packages/integration-tests/src/playbook/hardhat.config.ts:11]: We shouldn't do any file existence checks. We should just have a fallback to `Wallets.richWalletPrivateKey`.  No logic is needed here in this file except loading dotenv.

Comment (pcheremu) [reply packages/integration-tests/src/playbook/hardhat.config.ts]: @Romsters updated

File: packages/integration-tests/src/playbook/hardhat.config.ts [hunk 2]
```diff
@@ -4,10 +4,16 @@ require("dotenv").config({ path: __dirname + "/.env" });
 import "@matterlabs/hardhat-zksync-deploy";
 import "@matterlabs/hardhat-zksync-solc";
 import "@nomiclabs/hardhat-ethers";
+
 import { localConfig } from "../config";
+import { Wallets } from "../entities";
 
 import type { HardhatUserConfig } from "hardhat/types";
 
+if (!process.env.WALLET_PRIVATE_KEY) {
```
Comment (Romsters) [inline packages/integration-tests/src/playbook/hardhat.config.ts:13]: No need for this check and reassigning env vars is a bad pattern. I meant you can add a fallback in the place where it is used.
Like:

```
const wallet = new Wallet(process.env.WALLET_PRIVATE_KEY || Wallets.richWalletPrivateKey);
...
```

Comment (pcheremu) [reply packages/integration-tests/src/playbook/hardhat.config.ts]: Updated, thank you. @Romsters

**Review by Romsters — APPROVED** [review]

**Pull request merged**